client/stylesheets/_toCat.scss

```diff
@@ -2407,27 +2423,29 @@ i:hover {
     ul {
         margin: 0;
         padding: 0;
-        margin-top: 10px;
+        margin-top: 12px;
+        margin-left: -12px;
         display: none;
         position: absolute;
-        background-color: $color-spacegrey;
+        background-color: #404040 !important;
         // transition: all 0.1s linear;
         font-size: 0.8rem;
-        z-index: 1000;
+        z-index: 3;
+        border: 6px solid #404040;
 
         li {
-            margin: 0;
+            margin: 0 10px;
             color: #fff;
             text-align: left !important;
             width: 150px;
-            padding: 15px 10px 15px 0px;
-            border-bottom: 1px solid #555;
+            padding: 18px 50px 18px 0px;
+            border-top: 1px solid #555;
```

Comment:
Color literals like `#555` should only be used in variable declarations; they should be referred to via variable everywhere else.